### DISCUSSION TITLE
Replying to tweet with twitter-openapi-typescript?

### DISCUSSION THREAD

**Discussion by darkmasta** [fa0311/fa0311#3] [General]
How can I reply to a tweet with twitter-openapi-typescript? My failed attempts are below. An example of this in tests would be a great helper. 

```js
import login from './login.js';

const main = async () => {
    const { client, legacy } = await login();

    // works
    const uploadResult = await legacy.v1.uploadMedia('./test.jpg');
    console.log(uploadResult);

    // doesn't works
    // Uncaught Error: Request failed with code 403 - Sorry, you are not authorized to see this status. (Twitter code 179)
    const replyResult = await legacy.v1.reply('test', '1724038075507159140')
    console.log(replyResult);

    // doesnt creates reply
    const result  = await client.getPostApi().postCreateTweet({
        tweetText: 'Test reply...',
        mediaIds: [],
        extraParam: {
            "reply": {
                "in_reply_to_tweet_id": "1724038075507159140",
                "exclude_reply_user_ids": []
            },
        }
    });

};

main();
```

**Comment by fa0311** [comment]
Replying to tweets is not supported,
You can wait until it is supported or you can contribute to https://github.com/fa0311/twitter-openapi

**Comment by darkmasta** [comment]
Since I love your library I would really love to contribute. I will certainly do so in the future but in this particular case when I check the requests on twitter replying and creating have only 1 difference and its this field in the variables:

`"reply": {
                "in_reply_to_tweet_id": "1724038075507159140",
                "exclude_reply_user_ids": []
            },
`

When I add it to the `postCreateTweet` function params it should work. I have no other idea how to solve this. If you can give hints, and point me to the solution I can try even more, or else I have to wait for you to add this feature. Because I have already created an app with your library and I don't want to switch. :wink:

**Comment by fa0311** [comment]
@darkmasta 
I added this in v0.0.25
See the test. https://github.com/fa0311/twitter-openapi-typescript/blob/8fc0194ccc652400d29be0891e71ac2dfb59d923/twitter-openapi-typescript/test/api/postApi.test.ts#L11L20
